src/applications/appeals/10182/tests/pages/issueSummary.unit.spec.jsx

```diff
@@ -6,14 +6,16 @@ import sinon from 'sinon';
 import { DefinitionTester } from 'platform/testing/unit/schemaform-utils';
 
 import formConfig from '../../config/form';
-import { SELECTED, CONTESTABLE_ISSUES_PATH } from '../../constants';
+import { CONTESTABLE_ISSUES_PATH } from '../../constants';
```

Comment:
Would we want to keep the same pattern with `CONTESTABLE_ISSUES_PATH` to be `CONTESTED_ISSUES_PATH`?

Reply:
That's fair.